validator/keymanager/remote-web3signer/metrics.go

```diff
@@ -58,6 +60,13 @@ var (
 		Name: "remote_web3signer_blinded_block_deneb_sign_requests_total",
 		Help: "Total number of blinded block deneb sign requests",
 	})
+	/////
```

Comment:
Meant to be here?

Reply:
yeah just to show that it's deprecated to there